build/commands/lib/l10nUtil.js

```diff
@@ -258,16 +258,22 @@ const l10nUtil = {
     }
   },
 
-  // This simply reads Chromium files that are passed to it and replaces branding strings
-  // with Brave specific branding strings.
-  // Do not use this for filtering XML, instead use chromium-rebase-l10n.py.
-  // Only add idempotent replacements here (i.e. don't append replace A with AX here)
-  rebaseBraveStringFilesOnChromiumL10nFiles: async (path) => {
+  // This simply copies the content of Chromium files to their mapped Brave
+  // equivalents. Additionally, it removes GRDP files that are no longer in the
+  // Chromium GRD files.
+  rebaseBraveStringFilesOnChromiumL10nFiles: async () => {
     const removedMap = getRemovedGRDParts(getAutoGeneratedGrdMappings())
-    const ops = Object.entries(getChromiumToAutoGeneratedBraveMapping()).map(async ([sourcePath, destPath]) => {
-      let contents = await new Promise(resolve => fs.readFile(sourcePath, 'utf8', (err, data) => resolve(data)))
-      await new Promise(resolve => fs.writeFile(destPath, contents, 'utf8', resolve))
-    })
+    const ops = Object.entries(getChromiumToAutoGeneratedBraveMapping()).map(
+      async ([sourcePath, destPath]) => {
+        console.log("Resetting " + path.relative(srcDir, destPath) + " <- " +
+          path.relative(srcDir, sourcePath))
+        let contents = await new Promise(
+          resolve => fs.readFile(sourcePath, 'utf8',
+            (err, data) => resolve(data)))
+        await new Promise(
+          resolve => fs.writeFile(destPath, contents, 'utf8', resolve))
```

Comment:
do we really need to read/write manually?

The fs.copyFile() method is used to asynchronously copy a file from the source path to the destination path. By default, Node.js will overwrite the file if it already exists at the given destination.

Reply:
Nope, I don't think we do 😄